### DISCUSSION TITLE
Finding right FW versions

### DISCUSSION THREAD

**Discussion by kirk-the-jerk** [Jyers/Marlin#258] [Q&A] [answered]
1. How do I know if my Ender 3 v2 has v4.2.2 or 4.2.7 mainboard?
2. I'm currently using factory FW; want to upgrade the board FW to Jyer's pre-built version.  All I need is the pre-built FW from this github?:
3. ...or I also need to update the LCD firmware?  If yes, where can I find that; ideally pre-built also?

**Comment by clintiepoo** [comment, marked as answer]
1. You _probably_ have 4.2.2 unless you upgraded it. The way to know for sure is open the box up and look.
2. Yes, all you need is the pre-built firmware. If you're interested in customizing it yourself, you can also do that fairly easily, but it requires more.
3. The LCD firmware shouldn't need updated. I think there may be some new icon stuff coming, but it's not necessary to update the LCD firmware right now.
